### DISCUSSION TITLE
Cannot add a default Route using the Redirect component

### DISCUSSION THREAD

**Discussion by RubenZx** [remix-run/react-router#8976] [v5 to v6 Migration]
I started following the steps of the [official migration guide](https://github.com/remix-run/react-router/discussions/8753) and I got stuck in the **Migration Strategy** section. I have replaced all my routes (`Route` component from v5 package) with `CompatRoute`, without any problem, but when I try to replace my default redirect used for that routes that are not defined, I always get a blank page with no errors printed in console.

I have tried the following:
```tsx
import { Redirect } from 'react-router-dom';
import { CompatRoute } from 'react-router-dom-v5-compat';

// I have others redirects above the one that is not working that work fine, for example:
<CompatRoute
    exact
    path="/visitor"
    render={() => <Redirect exact to="/" />}
/>

// The following redirect doesn't work
<CompatRoute render={()=> <Redirect to="/" />} />
```

```tsx
import { Redirect } from 'react-router-dom';
import { CompatRoute } from 'react-router-dom-v5-compat';

// I have others redirects above the one that is not working that work fine, for example:
<CompatRoute
    exact
    path="/visitor"
    render={() => <Redirect exact to="/" />}
/>

// The following redirect doesn't work
<CompatRoute path="*" render={()=> <Redirect to="/" />} />
```

All my routes are wrapped by its corresponding `Switch` from '`react-router-dom`'. 

Am I doing anything wrong? Is there any problem with this?

**Comment by thejuan** [comment]
Adding `path=""` fixed for me
